extensions/wikia/Search/templates/WikiaSearch_index.php

```diff
@@ -1,15 +1,42 @@
 <section class="Search this-wiki WikiaGrid clearfix search-tracking">
-	<form class="WikiaSearch" id="search-v2-form" action="<?=$specialSearchUrl; ?>#">
+	<form class="WikiaSearch" id="search-v2-form" action="<?= $specialSearchUrl; ?>#">
 		<div class="SearchInput">
 			<?php if ( !empty( $advancedSearchBox ) ) : ?>
-				<p class="advanced-link"><a href="#" id="advanced-link"><?= wfMessage( 'searchprofile-advanced' ) ?></a></p>
-				<?php endif ?>
-
-				<p class="grid-1 alpha"><?= wfMsg( 'wikiasearch2-wiki-search-headline' ) ?></p>
+				<p class="advanced-link"><a href="#" id="advanced-link"><?= wfMessage( 'searchprofile-advanced' ) ?></a>
+				</p>
+			<?php endif ?>
 
-				<input type="text" name="search" id="search-v2-input" class="search-v2-input" value="<?=$query; ?>" />
-			<input type="hidden" name="fulltext" value="Search" />
-			<button type="submit" class="wikia-button" id="search-v2-button" value="<?= wfMsg( 'searchbutton' ); ?>"><img src="<?= $wg->BlankImgUrl ?>" class="sprite search" height="17" width="21"></button>
+			<div>
+				<div class="wds-dropdown">
+					<div class="wds-dropdown__toggle" style="border-right: 2px black">
```

Comment:
🛑 inline styles

Reply:
even if you need to set this border, then the color should probably match community theming instead of being hardcoded to black

Reply:
Looks like leftover - border is set in scss, thanks for catching that!